lib/iris/tests/unit/analysis/test_WPERCENTILE.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2015, Met Office
+# (C) British Crown Copyright 2015-2016, Met Office
```

Comment:
This needs to be formatted as 2015 - 2016 (note the spaces surrounding the dash -) as the current formatting is causing some of the license header tests to barf.